tests/posit/connect/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066.json

```diff
@@ -37,9 +37,24 @@
     "default_py_environment_management": null,
     "run_as": null,
     "run_as_current_user": false,
-    "owner_guid": "87c12c08-11cd-4de1-8da3-12a7579c4998",
     "content_url": "https://connect.example/content/f2f37341-e21d-3d80-c698-a935ad614066/",
     "dashboard_url": "https://connect.example/connect/#/apps/f2f37341-e21d-3d80-c698-a935ad614066",
     "app_role": "viewer",
-    "id": "8274"
-}
\ No newline at end of file
+    "id": "8274",
+    "owner_guid": "20a79ce3-6e87-4522-9faf-be24228800a4",
+    "owner": {
```

Comment:
Interestingly, the `GET v1/content/{guid}/` endpoint does *not* accept owner,tags as a query param, so this would not show up here: https://docs.posit.co/connect/api/#get-/v1/content/-guid-

Other fun detail I discovered on Friday: in `GET v1/content/?include=tags,owner`, while `owner` is always populated, `tags` is only added if there are any tags associated with the content; I expected to see `tags: []` but did not see that when inspecting real responses from Connect. 

I'd recommend copying the contents of the JSON from here: https://github.com/rstudio/connectapi/pull/223/files#diff-0ff270391c558938645b02da8e606b3e9bb14a131bf4d90a6ad21a2541871712 I was adding the API mocks we have in posit-sdk-py over there and had to add tags and owners, so that JSON there is consistent with the other mocks we have here, and it has some variation in whether or not tags exist.